playbooks/analytics_sandbox.yml

```diff
@@ -0,0 +1,35 @@
+- name: Bootstrap instance(s)
```

Comment:
@giovannicimolin, this needs to be updated based on the hawthorn version of the analytics playbook. See https://github.com/open-craft/openedx-deployment/commit/cca980973364bf1136640e5649dd93f31de26296#diff-40c660c4b60b66b30b41740d0dd05ea3 and https://github.com/hks-epod/configuration/pull/6 for details.

Reply:
@lgp171188 Can you check if I cherry-pick the correct changes?